addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -185,8 +168,24 @@ function factory(dependencies) {
          *
          * @param {MouseEvent} ev
          */
-        onClickStartAMeetingButton(ev) {
-            this.createMeetingChannel();
+        async onClickStartAMeetingButton(ev) {
+            const meetingChannel = await this.messaging.models['mail.thread'].createGroupChat({
+                default_display_mode: 'video_full_screen',
+                partners_to: [this.messaging.currentPartner.id],
+            });
+            if (!this.exists()) {
+                return;
+            }
+            await meetingChannel.open();
+            if (!meetingChannel.exists()) {
+                return;
+            }
+            await meetingChannel.toggleCall({ startWithVideo: true });
+            if (!this.threadView || !this.threadView.topbar) {
```

Comment:
condition on topbar no longer needed it seems

Reply:
Or actually isn't this method missing to open the invitation form now?

Reply:
Indeed, it doesn't open channel invitation form, will fix that